sample-app/src/main/java/com/joinforage/android/example/ui/complete/flow/balance/FlowBalanceFragment.kt

```diff
@@ -43,10 +44,19 @@ class FlowBalanceFragment : Fragment() {
             }
         }
 
-        foragePinEditText = binding.foragePinEditText
         snap = binding.snap
         nonSnap = binding.nonSnap
 
+        // as soon as possible set the forage context on
+        // the ForageElement
+        foragePinEditText = binding.foragePinEditText
+        foragePinEditText.setForageContext(
+            ForageContext(
+                merchantId = viewModel.merchantAccount,
+                sessionToken = viewModel.bearer
+            )
+        )
+
```

Comment:
I will absolutely do that in this PR. Or actually, I think it will be easier to do a follow on PR on top of this stack to clean this up. I'll ping once I've figured out the best way to do this

Reply:
Ohh I just realized that you're referring to the variables used in the implementation of `forage-sdk` and `smaple-app` and not fixing typos or case issues in the public-facing argument/interface names. 

In that case, I went ahead and made [a ticket](https://linear.app/joinforage/issue/FX-586/android-update-internal-merchantaccount-and-bearer-variables) for this request as I wholly agreed it needs. Good callout still!